src/agent/srcview/src/bin/srcview.rs

```diff
@@ -186,6 +197,7 @@ fn cobertura(opts: CoberturaOpt) -> Result<()> {
     let coverage: Vec<SrcLine> = modoffs
         .into_iter()
         .filter_map(|m| srcview.modoff(&m))
+        .flat_map(|sl| sl.map(|e| e.clone()))
         .collect();
 
     // Generate our report, filtering on our example path
```

Comment:
Small thing: just looking at `Report::new`, we could have it take an `impl Iterator<Item = &SrcLine>` which would avoid the need to instantiate a `Vec` here and also avoid cloning all the `SrcLine`s. It looks a little cleaner as well:

```rs
    // Convert our ModOffs to SrcLine so we can draw it
    let coverage = modoffs
        .into_iter()
        .filter_map(|m| srcview.modoff(&m))
        .flatten();

    // Generate our report, filtering on our example path
    let r = Report::new(coverage, &srcview, Some(r"E:\\1f\\coverage\\example")).unwrap();
```